reference/resources/bgpconfig.md

```diff
@@ -65,14 +65,14 @@ spec:
 
 | Field       | Description                 | Accepted Values   | Schema | 
 |-------------|-----------------------------|-------------------|--------|
-| name | Name or identifier for the community. | | string |
+| name | Name or identifier for the community. This should be used in [prefixAdvertisements](#prefixAdvertisements) to advertise the community value. | | string |
 | value | Standard or large BGP community value. |For standard community, value should be in `aa:nn` format, where both `aa` and `nn` are 16 bit integers.<br/> For large community, value should be `aa:nn:mm` format, where `aa`, `nn` and `mm` are all 32 bit integers. <br/>Where `aa` is an AS Number, `nn` and `mm` are per-AS identifier. | string |
   
 #### prefixAdvertisements
```

Comment:
I noticed that the anchor link doesn't match the ones in the markdown (`prefixadvertisements` vs `prefixAdvertisements`). 

So clicking on the links doesn't shift the page.